provider/schema/execution.go

```diff
@@ -134,6 +140,23 @@ func (e ExecutionData) truncateTable(ctx context.Context, client ClientMeta, par
 	return nil
 }
 
+// cleanupStaleData cleans resources in table that weren't update in the latest table resolve execution
+func (e ExecutionData) cleanupStaleData(ctx context.Context, client ClientMeta, parent *Resource) error {
+	// Only clean top level tables
+	if parent != nil {
+		return nil
+	}
+	if !e.disableDelete {
+		client.Logger().Debug("skipping stale data removal", "table", e.Table.Name)
```

Comment:
I suggest putting this in info just like line 153 is also info (or put 153 in debug).